en/get-started.md

```diff
@@ -581,6 +598,26 @@ Then access Grafana through <http://${remote-server-IP}:3000>.
 
 For more information about monitoring the TiDB cluster in TiDB Operator, refer to [Deploy Monitoring and Alerts for a TiDB Cluster](monitor-a-tidb-cluster.md).
 
+### Access TiDB Dashboard web UI
+
+You can forward the port for TiDB Dashboard to access TiDB Dashboard web UI locally:
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl port-forward -n tidb-cluster svc/basic-tidb-dashboard-exposed 12333 > pf12333.out &
+```
+
+You can access the panel of TiDB Dashboard at <http://localhost:12333> on the host where you run `kubectl`.
+
+Note that if you run `kubectl port-forward` in a Docker container or on a remote host instead of your local host, you can not access TiDB Dashboard using `localhost` from your browser. In this case, you can run the following command to listen on all addresses.
```

Comment:
```suggestion
Note that if you run `kubectl port-forward` in a Docker container or on a remote host instead of your local host, you cannot access TiDB Dashboard using `localhost` from your local browser. In this case, you can run the following command to listen on all addresses.
```